brightray/browser/inspectable_web_contents_impl.cc

```diff
@@ -412,8 +414,9 @@ void InspectableWebContentsImpl::CloseWindow() {
 
 void InspectableWebContentsImpl::LoadCompleted() {
   frontend_loaded_ = true;
-  if (managed_devtools_web_contents_)
-    view_->ShowDevTools();
+  if (managed_devtools_web_contents_) {
```

Comment:
Pretty sure we have a style guide / made a style decision that we wanted single liners to have no braces 🤔 

cc @ckerr

Reply:
I will change it, but IMHO single liners without braces are really bad idea. It reminds me of the goto fail bug

Reply:
We generally follow the Chrome / [Google C++ guidelines](https://google.github.io/styleguide/cppguide.html), where braces are generally optional (not forbidden) for single-liners:

>  In general, curly braces are not required for single-line statements, but they are allowed if you like them; conditional or loop statements with complex conditions or statements may be more readable with curly braces. Some projects require that an if must always have an accompanying brace.

So there's wiggle room there if we want to require braces or if we want them to be optional for single-liners. I don't really care what we use as long as we're consistent.

Reply:
> I don't really care what we use as long as we're consistent.

👍